Code of Conduct Iteration #1 (#16)
nelson-social/community: ExtendedDescription.md

## About nelson.social

Welcome to nelson.social, an online space dedicated to the community of Nelson, BC! We’re excited to build a place where people from all walks of life can come together to share ideas and passions, build connections, or just hang out and chat.

Here at nelson.social, we believe in the power of community and the importance of supporting one another in order to enrich our community. Whether you are a lifelong resident of Nelson, new to the region, or just passing through, we invite you to join the conversation and be a part of our community.

In this space, you can share your thoughts and ideas, ask questions of your neighbours, and even plan events and activities. You can also connect with people from all over the world, because nelson.social is part of the [Mastodon](https://github.com/mastodon/) network. We like to think of it like a big WhatsApp or iMessage group, but where you can choose who to share with and whose messages you see, even if they aren’t members of nelson.social.

> We encourage you to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. We encourage you to read them, and also reach out to us if you encounter any problems, or have any questions.

Reviewer: We use "encourage" twice here, can we use a stronger word?

Expect, demand, require?

Author: Sure, well spotted. I think this was copy @blaine used ChatGPT to generate!

Reviewer: ```suggestion
We expect our members to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. Please read them, and reach out to us if you encounter any problems, or have any questions.
```

We look forward to seeing you on nelson.social, and to building a vibrant and supportive community together. Welcome!

## Land Acknowledgement

> We would like to acknowledge that in Nelson we live and work on the traditional, ancestral and unceded territory of Sinixt Peoples. We make this acknowledgment to show our respect for the tmxʷulaʔxʷ (homeland), and [Sinixt](https://bloodoflifecollective.org/sinixt.html) Nation. We feel privileged and grateful to be here, and we invite those reading this to explore their own relationship to place.

Reviewer: @mattwynne are you happy with this?

IMO we can use it as is for now, but it lacks a call to action/accountability.

Author: I think it's good enough for now. Can we continue the discussion in #17?

Reviewer: Perfect.

## Code of Conduct

Nelson is home to people from all over the world who have a diverse set of skills, personalities, backgrounds and experiences. This diversity is wonderful, and we are committed to ensuring that the nelson.social community is a safe, enjoyable and thriving environment for the largest number possible of Nelson residents, their friends and family members.

> To that end, we will strive to make this a friendly, safe and welcoming environment for everyone regardless of qualities such as their gender identity or expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, nationality, ethnicity, socioeconomic status, family structure, spirituality, religion (or lack thereof), education, or other personal traits. We particularly celebrate diversity and do not tolerate bigotry or prejudice.

Reviewer: Love this.

Genuine question: Does it mean I can't joke about people who believe the earth is flat or that they are a mermaid with x-ray vision?

Author: I think that would be absolutely fine. Which of the traits on that list would you be worried that would come under? Religion?! 😀

Reviewer: I'm really just curious what radical inclusivity looks like in the real world. Maybe better to be discussed over a cuppa 🇬🇧

> As a community, we expect any participant to act according to the following ground rules. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.

Reviewer: The reference of "following ground rules", what does it refer to? The next sentence or "guiding principle"?

Author: Yeah we should be more explicit and say "guidelines in this document" or some such. Good catch!

Reviewer: ```suggestion
As a community, we expect any participant to act according to the guidelines in this document. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.
```

This is a living document, and the nelson.social team welcome your feedback. We will amend this document as necessary to ensure the safety of our users.

### Guiding principle

*If you wouldn't do it in Oso Negro, don't do it on nelson.social*

### Handling conflict

A diverse group of people has a diverse set of boundaries about what is OK. Sometimes one of us may make a mistake, and do something that inadvertently causes offence. When we observe this happen, we try to give the offender feedback quickly, clearly and constructively. We try to offer them the opportunity to maintain their dignity in the situation, to learn from and repair their mistake.

This kind of learning and repair is how we strengthen our relationships and our community. However we will not tolerate anyone deliberately or repeatedly breaking these rules.

### Consequences of Unacceptable Behavior

Unacceptable behavior from any community member, including those with decision-making authority, will not be tolerated.

Anyone asked to stop unacceptable behavior is expected to comply immediately.

> If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.

Reviewer: Can we say:

"temporary suspension or permanent ban"?

Those would be the mastodon terms to avoid confusion and use DSL.

Reviewer: ```suggestion
If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary suspension or permanent ban from the community without warning.
```

### Reporting Guidelines

> If you are subject to or witness unacceptable behavior, or have any other concerns, please [notify a community organizer](#team) by emailing [admin@nelson.social](mailto:admin@nelson.social) as soon as possible. We are here to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.

Reviewer: "Notify a community organizer via direct message, or email admin@nelson.social"

Maybe that to make it less formal to always use email? It would be quicker for us as well to react to.

Reviewer: Maybe we should also mention that people can "report" toots directly (on mobile and web app).

Reviewer: Maybe there is a level 1 "please take a look at this, it offensive" and a level 2 "DM/email the team"?

Since mastodon has moderation/reporting built in we may as well use it.

Author: I thought about that but don't know mastodon well enough yet to be able to describe it. It would be great if you want to try and add some detail here.

Reviewer: ```suggestion
If you are subject to or witness unacceptable behavior, please use the "Report user" functionality of Mastodon. If the situation is beyond unacceptable behavior, please notify any of the [Team](#team) via direct message. We are here to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.
```

* A place for general advertisement. Promoting your original work is welcome, but please avoid overly commercial self-promotion

You may [find another server](https://instances.social/) to be a better fit if you don't like these rules.

### Encouraged Behavior

The following behaviors are expected and requested of all community members:

* Be friendly, open and welcoming.
* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
* Exercise consideration and respect in your speech and actions.
* Attempt collaboration & de-escalation before conflict.
* If you inadvertently cause offence to someone else, take responsibility for your actions.
* Refrain from demeaning, discriminatory, or harassing behavior and speech.
* Be mindful of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
  
### Unacceptable Behavior

To be specific, the following behaviors are unacceptable within our community:

> * Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in Canada.

Reviewer: I just spend a week doing legal. We should say Canada or British Columbia, since that's where to court would take place.

Reviewer: ```suggestion
* Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in British Columbia or Canada.
```

This code of conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.

### License and attribution

The nelson.social Code of Conduct is distributed by [nelson.social](http://nelson.social) under a [Creative Commons Attribution-ShareAlike license](http://creativecommons.org/licenses/by-sa/3.0/). 

Portions of text derived from:

* [Django Code of Conduct](https://www.djangoproject.com/conduct/)
* [Geek Feminism Anti-Harassment Policy](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy)
* [Social.coop Code of Conduct](https://wiki.social.coop/rules-and-bylaws/Code-of-conduct.html)
* [Fosstodon Code of Conduct](https://hub.fosstodon.org/coc/)
* [Ottawa.place Code of Conduct](https://ottawa.place/terms#coc)
* [Cucumber Community Code of Conduct](https://cucumber.io/conduct)

## Team

This server is hosted and run by a team of volunteers who live here in Nelson, BC:

> * [@blaine](https://nelson.social/web/@blaine)
> * [@matt](https://nelson.social/web/@matt)
> * [@till](https://nelson.social/web/@till)

Reviewer: Let's use standard links that work anywhere.

```suggestion
* [@blaine](https://nelson.social/@blaine)
* [@matt](https://nelson.social/@matt)
* [@till](https://nelson.social/@till)
```
